### ISSUE TITLE
The type 'double precision' generates mermaid code with invalid syntax.

### ISSUE THREAD

**Issue by takahashi-sdi** [imjuni/erdia#57]
Hello,

I have an entity that includes a double precision type.

```
@Entity()
export class SomeEntity {
  @PrimaryGeneratedColumn('uuid', { comment: 'ID' })
  readonly id!: string;

  @Column({ type: 'double precision', comment: 'Latitude' })
  lat!: number;

  @Column({ type: 'double precision', comment: 'Longitude'  })
  lon!: number;
}
```

Erdia generates mermaid source code as follows.

```
"some_entities(SomeEntity)" {
  uuid id    PK      "ID"  
  double precision lat      "Latitude"  
  double precision lon      "Longitude"  
```

This code syntax is invalid because mermaid does not support the double precision type.

**Comment by imjuni** [comment]
This looks like an issue with some formats supported by typeorm that are not mysql DB data types. For example, the same issue can occur with simple-json other than double precision. I'll see if I can find a way to fix it, or feel free to send me a PR with your own fix.

**Comment by imjuni** [comment]
@takahashi-sdi Fixed [`3.4.1`](https://github.com/imjuni/erdia)
